### PR TITLE
Update commands.md with /cc commands

### PR DESCRIPTION
**Pull request by cblecker**
Update commands.md with the /cc commands from #2513

### CODE REVIEW

File: commands.md [hunk 1]
```diff
@@ -4,8 +4,10 @@
 
 Command | Implemented By | Who can run it | Description
 --- | --- | --- | ---
-`/assign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | kubernetes org members | Assigns specified people (or yourself if no one is specified)
-`/unassign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | kubernetes org members | Unassigns specified people (or yourself if no one is specified)
+`/assign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Assigns specified people (or yourself if no one is specified). Target must be a kubernetes org member.
+`/unassign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Unassigns specified people (or yourself if no one is specified). Target must already be assigned.
+`/cc @userA [@userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Request review from specified people. Target must be a kubernetes org member.
```
Comment (fejta) [inline commands.md]: Note this will change to making @userA optional in the very near future :happy:

**Comment by fejta** [comment]
LGTM will let Cole do the official review

**Comment by cjwagner** [comment]
/lgtm
I'll update the commands.md file to indicate that the bare '/cc' command without any users is acceptable in #2626

**Pull request merged**